src/EditorFeatures/Core/Shared/Threading/AsynchronousSerialWorkQueue.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Threading
     //
     // 1) Background actions are run serially.  This allows you to enqueue a whole host of background
     // work to do, without having to worry about those same background tasks running simultaneously 
-    // and mungeing each other inappropriately.
+    // and munging each other inappropriately.
```

Comment:
Urban dictionary says this word indicates a vulgar act (so it would almost certainly be considered "inappropriate"). Consider: "interfering with" or "colliding with".

"inappropriately" could probably be dropped.